rskj-core/src/main/java/co/rsk/peg/constants/BridgeConstants.java

```diff
@@ -98,7 +97,13 @@ public String getBtcParamsString() {
         return btcParamsString;
     }
 
-    public Federation getGenesisFederation() { return genesisFederation; }
+    public List<BtcECKey> getGenesisFederationPublicKeys() {
+        return genesisFederationPublicKeys;
+    }
+
+    public Instant getGenesisFederationAddressCreatedAt() {
+        return genesisFederationAddressCreatedAt;
```

Comment:
```suggestion
    public Instant getGenesisFederationCreationTime() {
        return genesisFederationCreationTime;
```
getCreatedAt sounds a bit weird

Also, it's the federation creation time, not the address creation

Reply:
Done!